pkgs/applications/audio/spotify-player/default.nix

```diff
@@ -60,7 +60,7 @@ rustPlatform.buildRustPackage rec {
     ++ lib.optionals withSixel [ libsixel ]
     ++ lib.optionals (withAudioBackend == "alsa") [ alsa-lib ]
     ++ lib.optionals (withAudioBackend == "pulseaudio") [ libpulseaudio ]
-    ++ lib.optionals (withAudioBackend == "rodio") [ alsa-lib ]
+    ++ lib.optionals (withAudioBackend == "rodio" && stdenv.isLinux) [ alsa-lib ]
```

Comment:
It looks like the change should be made to `withAudioBackend? "rodio"` instead of here, so that `rodio` isn't set by default on darwin.

Reply:
@wegank I'm using rodio successfully in Darwin, so I don't think rodio should be excluded as an option.